frontend/src/summary.pug

```diff
@@ -122,11 +122,19 @@
           title="Total contribution of group"
         ) [{{ getGroupTotalContribution(repo) }} lines]
         template(v-if="isMergeGroup")
+          a(
+            v-if="filterGroupSelection === 'groupByRepos'",
+            title="click to view repo's code",
+            onclick="deactivateAllOverlays()",
+            v-on:click="openTabAuthorship(repo[0], repo, i)"
```

Comment:
Can't you just pass in a `0` here for the `index` parameter, instead of modifying the `openTabAuthorship` method? Seems like the argument `i` is not used in `openTabAuthorship`.

Reply:
Yep, didn't realise that